### PR TITLE
refactor(Core/BattlegroundRV): Convert to std::chrono

### PR DESCRIPTION
**Pull request by Kitzunu**
## Changes Proposed:

* setTimer -> SetTimer

* getTimer -> GetTimer

* SetTimer && GetTimer now use Milliseconds

## Issues Addressed:

- ref https://github.com/azerothcore/azerothcore-wotlk/issues/6490

## Tests Performed:

- Builds

## How to Test the Changes:

1. Check that all the timers in RV works the same as before

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Kitzunu** [comment]
@azerothcore/testers

File: src/server/game/Battlegrounds/Zones/BattlegroundRV.h [hunk 1]
```diff
@@ -77,15 +77,16 @@ enum BattlegroundRVData
...
-    BG_RV_FIRST_TIMER                            = 20500, // elevators rise in 20133ms
     BG_RV_WORLD_STATE_A                          = 0xe11,
     BG_RV_WORLD_STATE_H                          = 0xe10,
     BG_RV_WORLD_STATE                            = 0xe1a,
 };
 
+static constexpr Milliseconds BG_RV_PILLAR_SWITCH_TIMER  = 25s;
+static constexpr Milliseconds BG_RV_FIRE_TO_PILLAR_TIMER = 20s;
+static constexpr Milliseconds BG_RV_CLOSE_FIRE_TIMER     = 5s;
+static constexpr Milliseconds BG_RV_FIRST_TIMER          = 20500ms; // elevators rise in 20133ms
```
Comment (Winfidonarleyan) [inline src/server/game/Battlegrounds/Zones/BattlegroundRV.h:88]: maybe move this in source file `.cpp` ?

Comment (Kitzunu) [reply src/server/game/Battlegrounds/Zones/BattlegroundRV.h:88]: I can do it, just want to learn why?

Comment (Winfidonarleyan) [reply src/server/game/Battlegrounds/Zones/BattlegroundRV.h:88]: I think the less information in the headers, the better

Comment (Kitzunu) [reply src/server/game/Battlegrounds/Zones/BattlegroundRV.h:88]: done

File: src/server/game/Battlegrounds/Zones/BattlegroundRV.cpp [hunk 2]
```diff
@@ -25,6 +25,11 @@
 #include "WorldPacket.h"
 #include "WorldSession.h"
 
+static constexpr Milliseconds BG_RV_PILLAR_SWITCH_TIMER  = 25s;
+static constexpr Milliseconds BG_RV_FIRE_TO_PILLAR_TIMER = 20s;
+static constexpr Milliseconds BG_RV_CLOSE_FIRE_TIMER     = 5s;
+static constexpr Milliseconds BG_RV_FIRST_TIMER          = 20500ms; // elevators rise in 20133ms
```
Comment (Winfidonarleyan) [inline src/server/game/Battlegrounds/Zones/BattlegroundRV.cpp:31]: why static?

**Comment by Winfidonarleyan** [comment]
- [x] Work

**Pull request merged**